### PR TITLE
Fix domain for juntagrico 1.5

### PR DESCRIPTION
**Pull request by nixnuex**

### CODE REVIEW

**Review by samuel-iseli — CHANGES_REQUESTED** [review]
I would include protocol in serverUrl. The change in the text template isn't needed then. Otherwise the translations would need to be adapted too.

**Comment by nixnuex** [comment]
Done

File: juntagrico_billing/mailer.py [hunk 1]
```diff
@@ -13,8 +13,7 @@ def send_bill_share(bill, share, member):
         'member': member,
         'bill': bill,
         'share': share,
-        'protocol': 'https',
-        'serverurl': Site.objects.get_current().domain
+        'serverurl': 'https' + Site.objects.get_current().domain
```
Comment (samuel-iseli) [inline juntagrico_billing/mailer.py:16]: Isn't the `://` missing now? 
Should be `'https://' + Site.objects.get_current().domain` to yield the same result as before.

**Review by samuel-iseli — CHANGES_REQUESTED** [review]

**Comment by nixnuex** [comment]
Sorry, sure.

**Review by samuel-iseli — APPROVED** [review]

**Comment by nixnuex** [comment]
Obsolete

**Pull request closed without merging**